app/models/skill.rb

```diff
@@ -0,0 +1,3 @@
+class Skill < ApplicationRecord
```

Comment:
please document somewhere how exactly the skill table will add to functionality in the future (we will use this for our readme)

Reply:
Added a comment on line 3 of `skill.rb`